lisa/util/constants.py

```diff
@@ -134,3 +134,7 @@
 DEVICE_TYPE_SRIOV = "SRIOV"
 DEVICE_TYPE_NVME = "NVME"
 DEVICE_TYPE_GPU = "GPU"
+
+
+# Remote Server address
+REMOTE_SERVER = "8.8.4.4"
```

Comment:
The name is confusing. How about `PINGABLE_INTERNET_SERVER`? And you can add some comments to say it's used to check internet connection